common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java

```diff
@@ -81,6 +78,18 @@ public static void setSystemProperties(String spanEndpoint,
       otelResourceAttributes += "," + resourceAttributes;
     }
     System.setProperty("otel.resource.attributes", otelResourceAttributes);
+
+    if (logLevel != null) {
+      System.setProperty("io.opentelemetry.auto.slf4j.simpleLogger.defaultLogLevel", logLevel);
```

Comment:
In a follow up, let's define these values as constants, maybe in its own class (maybe something like `Constants.java` with inner classes, in order to have `Constants.OpenTelemetry.LoggingLevelKey` and `Constants.OTLP.Etc`).